.travis.gogenerate.sh

```diff
@@ -1,10 +1,13 @@
 #!/bin/bash
 
-if [[ "$TRAVIS_GO_VERSION" =~ ^1\.[45](\..*)?$ ]]; then
+# If GOMOD is defined we are running with Go Modules enabled, either
+# automatically or via the GO111MODULE=on environment variable. Codegen only
+# works with modules, so skip generation if modules is not in use.
+if [[ -z "$(go env GOMOD)" ]]; then
```

Comment:
Won't this make all of our travis tests which specify `GOMODULE=off` pointless?

Reply:
The generated code already exists, so I would imagine tests would pass still. We wouldn't catch situations where generate needed to be run and wasn't, but other builds should catch that. Maybe?

Reply:
You're right, generated code would already exist just when modules if off we aren't checking that it regenerates the same. This is fine I think. The codegen code isn't worth trying to make work on both modules and not modules.